include/clang/Interpreter/CppInterOp.h

```diff
@@ -173,6 +173,9 @@ namespace Cpp {
   /// Checks if the passed value is an enum type or not.
   bool IsEnumType(TCppType_t type);
 
+  /// Returns vector of enums present in given scope
+  std::vector<std::string> GetEnums(TCppScope_t scope);
```

Comment:
I am wondering where is the implementation of `GetEnums`?

Reply:
Its already implemented in CppInterOp.cpp